Changelogs for Digital Twin Manager UI 1021.1.0 (#2916)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-1-0-removed-duplicate-buttons-from-cockpit-and-device-management-ctm-1795.md

---
date: ""

> title: "Fixed an issue where navigation elements were duplicated when using the official plugin: Asset Properties widget "

Reviewer: ```suggestion
title: Asset properties widget plugin no longer causes duplicate navigation elements
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1782,CTM-1795"
version: "1021.1.0"
---

> Fixed a user interface issue where navigation elements appeared twice after installing the Asset Properties widget plugin. This issue caused duplicate menu items, such as "Add Dashboard" in the Cockpit application and "Home" in Device Management. The fix ensures navigation elements appear only once, improving the user interface clarity and navigation experience.

Reviewer: ```suggestion
Fixed a user interface issue where navigation elements appeared twice after installing the "Asset properties" widget plugin. This issue caused duplicate menu items, such as **Add dashboard** in the Cockpit application and **Home** in the Device Management application. The fix ensures navigation elements appear only once, improving the user interface clarity and navigation experience.
```

---
date: ""

> title: "Removed duplicate buttons from Cockpit and Device Management"

Reviewer 2: ```suggestion
title: "Removed duplicated buttons from Cockpit and Device Management"
```

Reviewer: Cockpit and Device Management > Cockpit and Device Management application

Reviewer 3: Reading this title it seems the bug was in the cockpit and DM & not in the widget
```suggestion
title: "Fixed an issue where navigation elements were duplicated when using the official plugin: Asset Properties widget "
```

build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1782,CTM-1795"
version: "1021.1.0"
---

> Previously, in Device Management, there were two **Home** buttons displayed. In the Cockpit application, clicking on any asset resulted in a duplicate **Add Dashboard** button. These issues have now been fixed to avoid confusion for users.

Reviewer 2: ```suggestion
Previously, the **Home** button in the Device Management application and **Add Dashboard** button in the Cockpit application were displayed twice upon installing the Add asset plugin. These have now been resolved to avoid confusion for users.
```

Reviewer 3: ```suggestion
Fixed a user interface issue where navigation elements appeared twice after installing the Asset Properties widget plugin. This issue caused duplicate menu items, such as "Add Dashboard" in the Cockpit application and "Home" in Device Management. The fix ensures navigation elements appear only once, improving the user interface clarity and navigation experience.
```
